qa/logging-config/custom-log4j2.properties

```diff
@@ -7,7 +7,7 @@ logger.action.level = debug
 
 appender.rolling.type = RollingFile
 appender.rolling.name = rolling
-appender.rolling.fileName = ${sys:es.logs.base_path}${sys:file.separator}${sys:es.logs.cluster_name}_server.json
+appender.rolling.fileName = ${sys:es.logs.base_path}${sys:file.separator}${sys:es.logs.cluster_name}_server.log
```

Comment:
I think the `_server` suffix needs to be removed as well for the `.log` file?

Reply:
this is a test file. I intentionally keep this file slightly different